Core/CMakeLists.txt

```diff
@@ -111,3 +111,9 @@ add_subdirectory(src/Utilities)
 add_subdirectory(src/Vertexing)
 add_subdirectory(src/Visualization)
 add_subdirectory(src/AmbiguityResolution)
+
+# Help with compiler flags discovery
+include(ActsFunctions)
```

Comment:
out of curiosity: why is this necessary?

Reply:
I think it needs to be included somewhere once for the function to be available. I guess this location works because it's evaluated early on. I'll move it out to the top level `CMakeLists.txt` however.